README.md

```diff
@@ -270,13 +139,13 @@ I would like to thank...
 
 > Add at least 2 questions new developers would ask when they decide to use your project.
```

Comment:
**OPTIONAL** - Please remove this and all placeholders present in your readme, it'd help make your readme more professional

Reply:
Sure